### PR TITLE
[Backlog-36850] Focus Visible - Accessibility -  PUC Home Page – Manage Data Sources

### PR DESCRIPTION
**Pull request by varuntangirala**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **333**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-830/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 14s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **333**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-830/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
  * See the GNU Lesser General Public License for more details.
  *
- * Copyright (c) 2002-2023 Hitachi Vantara. All rights reserved.
+ * Copyright (c) 2002-2023 Hitachi Vantara..  All rights reserved.
```
Comment (dcleao) [inline widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java:15]: Please, remove the added, second ".".

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 2]
```diff
@@ -648,6 +653,15 @@ input#cron-tb {
   margin-bottom: 15px;
 }
 
+:root {
```
Comment (dcleao) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:656]: Should be placed near the beginning of the file (after font stuff).

**Review by dcleao — CHANGES_REQUESTED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 49s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **333**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-830/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Pull request merged**